<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mPsoBaseWdt</class>
 <widget class="QWidget" name="mPsoBaseWdt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="10">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Delay1</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QSpinBox" name="mYs1Edit">
              <property name="minimumSize">
               <size>
                <width>210</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>180</width>
                <height>50</height>
               </size>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="maximum">
               <number>5000</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>Delay2</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QSpinBox" name="mYs2Edit">
              <property name="minimumSize">
               <size>
                <width>210</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>180</width>
                <height>50</height>
               </size>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="maximum">
               <number>5000</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QGroupBox" name="groupBox_7">
           <property name="title">
            <string>Delay3</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <widget class="QSpinBox" name="mYs3Edit">
              <property name="minimumSize">
               <size>
                <width>210</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>180</width>
                <height>50</height>
               </size>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="maximum">
               <number>5000</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox_8">
           <property name="title">
            <string>Delay4</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <widget class="QSpinBox" name="mYs4Edit">
              <property name="minimumSize">
               <size>
                <width>210</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>180</width>
                <height>50</height>
               </size>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="maximum">
               <number>5000</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QGroupBox" name="groupBox_9">
           <property name="title">
            <string>Delay5</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0">
             <widget class="QSpinBox" name="mYs5Edit">
              <property name="minimumSize">
               <size>
                <width>210</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>180</width>
                <height>50</height>
               </size>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="maximum">
               <number>5000</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Pulse duration2</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QSpinBox" name="mMc2Edit">
              <property name="minimumSize">
               <size>
                <width>210</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>180</width>
                <height>50</height>
               </size>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="maximum">
               <number>5000</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Divider0</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QSpinBox" name="mFenPin1">
              <property name="minimumSize">
               <size>
                <width>210</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>180</width>
                <height>50</height>
               </size>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="minimum">
               <number>2</number>
              </property>
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Divider1</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QSpinBox" name="mFenPin2">
              <property name="minimumSize">
               <size>
                <width>210</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>180</width>
                <height>50</height>
               </size>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="minimum">
               <number>2</number>
              </property>
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Divider2</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QSpinBox" name="mFenPin3">
              <property name="minimumSize">
               <size>
                <width>210</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>180</width>
                <height>50</height>
               </size>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="minimum">
               <number>2</number>
              </property>
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
